genologics/config.py

```diff
@@ -4,28 +4,57 @@
 
 import ConfigParser
 
-config = ConfigParser.SafeConfigParser()
-try:
-	conf_file = config.read([os.path.expanduser('~/.genologicsrc'), '.genologicsrc',
-				'genologics.conf', 'genologics.cfg', '/etc/genologics.conf'])
-
-	# First config file found wins
-	config.readfp(open(conf_file[0]))
-	
-	BASEURI = config.get('genologics', 'BASEURI').rstrip()
-	USERNAME = config.get('genologics', 'USERNAME').rstrip()
-	PASSWORD = config.get('genologics', 'PASSWORD').rstrip()
-
-except:
-	warnings.warn("Please make sure you've created your own Genologics configuration file (i.e: ~/.genologicsrc) as stated in README.md")
-	sys.exit(-1)
-
-if config.has_section('genologics') and config.has_option('genologics','VERSION'):
-	VERSION = config.get('genologics', 'VERSION').rstrip()
-else:
-	VERSION = 'v2'
-	
-if config.has_section('logging') and config.has_option('logging','MAIN_LOG'):
-	MAIN_LOG = config.get('logging', 'MAIN_LOG').rstrip()
-else:
-	MAIN_LOG = None
+'''
+Usage:
+from genologics.config import BASEURI, USERNAME, PASSWORD
+
+Alternate Usage: 
+from genologics import config
+BASEURI, USERNAME, PASSWORD, VERSION, MAIN_LOG = load_config(specified_config = <path to config file>) 
```

Comment:
Don't you mean 
`BASEURI, USERNAME, PASSWORD, VERSION, MAIN_LOG = config.load_config(specified_config = <path to config file>)` ?
By using `from genologics import config` , you only get `config` in the namespace, not all the declared functions.